src/plugins/application_config/server/index.ts

```diff
@@ -20,4 +20,8 @@ export function plugin(initializerContext: PluginInitializerContext) {
   return new ApplicationConfigPlugin(initializerContext);
 }
 
-export { ApplicationConfigPluginSetup, ApplicationConfigPluginStart } from './types';
+export {
+  ApplicationConfigPluginSetup,
```

Comment:
this code pretty/format change should not be part of csp PR right?

Reply:
It adds a type `ConfigurationClient,` to the list. Thus it is not just format change.

Since the new plugin needs to import this from ApplicationConfigPlugin, lint checker will complain if this is not exposed at `index.ts` level. See the following lint error for reference.


```

ERROR 
      /home/ubuntu/workspace/clickjacking/osd-dev/OpenSearch-Dashboards/src/plugins/csp_handler/server/csp_handlers.ts
        6:37  error  Unexpected path "../../application_config/server/types" imported in restricted zone. Plugins may only import from top-level public and server modules  @osd/eslint/no-restricted-paths
      
      ✖ 1 problem (1 error, 0 warnings)
      
ERROR [eslint] errors
```